service/mongodb/controller.go

```diff
@@ -0,0 +1,100 @@
+package mongodb
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/meateam/fav-service/service"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	pb "github.com/meateam/fav-service/proto"
+
+)
+
+// Controller is the favorite service business logic implementation using MongoStore.
+type Controller struct {
+	store MongoStore
+}
+
+func NewMongoController(db *mongo.Database) (Controller, error) {
+	store, err := newMongoStore(db)
+	if err != nil {
+		return Controller{}, err
+	}
+
+	return Controller{store: store}, nil
+}
+
+func (c Controller) GetAllFavorites(ctx context.Context, userID string) ([]*pb.FavoriteObject, error) {
```

Comment:
DOCUMENTATION,
But I'll let the linter yell at you after you configure it

Reply:
done